HW7(Tests)/Functions.test.js

```diff
@@ -0,0 +1,167 @@
+const fetch = require('node-fetch'); // implementing fetch in Node.js
+const {describe, test, expect, beforeAll} = require('@jest/globals');
+const {validateEmail, User, getUsers, getUsersEmails} = require('./Functions.js');
+
+describe('Testing function validateEmail with different emails.', () => {
+  describe('Should return true with valid emails.', () => {
+    const validEmailsToCheck = [
+      'example@gmail.com',
+      'EXAMPLE@GMAIL.COM',
+      'ExAmPlE@mail.ru',
+      '        example@gmail.com         ',
+      '        example@gmail.com',
+      'example@gmail.com         ',
+      'exmpl@mail.ru',
+      'example12@gmail.com',
+      '12example@gmail.com',
+    ];
+
+    test('Returns true for valid emails.', () => {
+      for (const email of validEmailsToCheck) {
+        expect(validateEmail(email)).toBe(true);
+      }
+    });
+  });
+
+  describe('Should return false with invalid emails.', () => {
+    const invalidEmailsToCheck = [
+      'exmp@gmail.ru',
+      'exm@mail.ru',
+      'example @gmail.com',
+      'example@gmail. com',
+      'example@mailru',
+      'exmple@mailru.',
+      '.example@mailru',
+      'exm@.mailru',
+      'exm@.',
+      'examplemail.ua',
+      '@examplemail.ua',
+      'examplemail.ua@',
+      'example@12gmail.com',
+      'example@gmail.12',
+      'ex!ample@gmail.com',
+      'ex?ample@gmail.com',
+      'ex,ample@gmail.com',
+      'ex@..a@mple@gmail.com',
+      'ex.ample@gmail.com'
+    ];
```

Comment:
not critical, but for convenience, it is better to store data like this separately from the test itself. for example at the beginning of a file.